binding.cc

```diff
@@ -121,7 +121,7 @@ class Encoder : public Nan::ObjectWrap
 
     // Build cookie buffer.
     uint32_t cookieSize = e->enc_.GetMagicCookieSize(e->outf_.mChannelsPerFrame);
-    char cookie[cookieSize];
+	char* cookie = (char*)alloca(sizeof(char) * (cookieSize));
```

Comment:
I solved the same problems (building for windows).

I would prefer a ifndef since for gcc it doesn't seem to be a problem.

See my changes here:
https://github.com/m-a-v/node-libalac/commits/master